microsoft-365/compliance/ediscovery-troubleshooting-common-issues.md

```diff
@@ -198,6 +198,26 @@ When downloading search results using the eDiscovery Export Tool, it's possible
 
 To resolve this issue, retry [downloading the search results](export-search-results.md#step-2-download-the-search-results), which will restart the eDiscovery Export Tool.
 
+## Error/issue: Downloaded export shows no results
+
+After a successful export, the completed download via the export tool shows zero files in the results.
+
+### Resolution
+
+This is a client-side issue and in order to remediate it, please attempt the following steps:
+
+1. Try using another client/machine to download.
+
+2. Make sure to download to a local drive.
+
+3. Make sure the virus scanner is not running.
+
+4. Make sure that no other export is downloading to the same folder or any parent folder.
+
+5. If the previous steps did not work, disable zipping and de-duplication.
+
 ## See Also
 
 - [Tips to avoid content location errors](retry-failed-content-search.md#tips-to-avoid-content-location-errors)
+
+6. If this works then the issue is due to a local virus scanner or a disk issue.
```

Comment:
```suggestion
6. If this works then the issue is due to a local virus scanner or a disk issue.
## See Also

- [Tips to avoid content location errors](retry-failed-content-search.md#tips-to-avoid-content-location-errors)

```